localization/english/culture/wc_culture_titles_l_english.yml

```diff
@@ -1,29 +1,323 @@
 ﻿l_english:
+ ####################################################
+ # Special titles
+ ### Places ###
+ city_ahn_qiraj:0 "City"
+ quarter_grim_batol:0 "Quarter"
+ city_grim_batol:0 "City"
 
-####################################################
-#Special titles
+ ### Orcs ###
+ warchief_male:0 "Warchief"
+ warchief_female:0 "Warchief"
+
+ ### Night Elves ###
  high_priest_kaldorei_male:0 "$HIGH_PRIEST_MALE_NAME$"
  high_priestess_kaldorei_female:0 "$HIGH_PRIEST_FEMALE_NAME$"
+
+ # Special cultural titles (sorted by culture, then government type).
+ ### Orcs ###
+ # Feudal
+ count_feudal_male_orc:0 "Overlord"
+ count_feudal_female_orc:0 "Overlord"
+ duke_feudal_male_orc:0 "High Overlord"
+ duke_feudal_female_orc:0 "High Overlord"
+ king_feudal_male_orc:0 "Warlord"
+ king_feudal_female_orc:0 "Warlord"
+ emperor_feudal_male_orc:0 "Warchief"
+ emperor_feudal_female_orc:0 "Warchief"
+
+ ### High Elves / Blood Elves ###
+ # Feudal
+ count_feudal_male_highelf:0 "Lord"
+ count_feudal_female_highelf:0 "Lady"
+ county_feudal_highelf:0 "Lordship"
+ emperor_feudal_male_highelf:0 "High King"
+ emperor_feudal_female_highelf:0 "High Queen"
+ empire_feudal_highelf:0 "High Kingdom"
+
+ ### Night Elves ###
+ # Feudal
+ duke_feudal_male_nightelf:0 "Governor"
+ duke_feudal_female_nightelf:0 "Governor"
+ duchy_feudal_nightelf:0 "Province"
+ king_feudal_male_nightelf:0 "Prince"
+ king_feudal_female_nightelf:0 "Princess"
+ kingdom_feudal_nightelf:0 "Principalty"
```

Comment:
Night elves called princes? Not highborne?

Reply:
I do not understand

Reply:
> I do not understand

Why their kings are called Princes?

Reply:
done in b419ddbb1 too